src/IterableCodeExtractor.php

```diff
@@ -233,7 +233,7 @@ function ( $file, $key, $iterator ) use ( $include, $exclude, $extensions ) {
 
 			$filtered_files[] = Utils\normalize_path( $file->getPathname() );
 		}
-
+		sort( $filtered_files );
```

Comment:
Should this use the `SORT_NATURAL` flag?

Reply:
Or then perhaps [natsort](https://www.php.net/manual/en/function.natsort.php) could be used? As long as the sorting behaves the same way on all systems without taking e.g. different system locales into consideration, then https://github.com/wp-cli/i18n-command/issues/171 will be resolved regardless of whether the files are sorted in natural order or not. Would sorting in natural order provide some additional benefit?

Reply:
`SORT_NATURAL` or `natsort` don't take locales into account, so that should not be a problem.

And yes, this PR seems to already solve #171. I was just wondering whether it would make sense to have more natural sorting. Examples from the docs page:

```
Standard sorting
Array
(
    [3] => img1.png
    [1] => img10.png
    [0] => img12.png
    [2] => img2.png
)

Natural order sorting
Array
(
    [3] => img1.png
    [2] => img2.png
    [1] => img10.png
    [0] => img12.png
)
```

The latter makes more sense to me there :-)

Reply:
Added `SORT_NATURAL` and `SORT_FLAG_CASE` flags (PHP>=5.4.0) to `sort` so that the sorting order is more natural :). Did not use `natsort` because it maintains key-value associations which is not necessary when having numerical indices.